reactive/reactive.js

```diff
@@ -120,6 +120,7 @@ module('users.timfelgentreff.reactive.reactive').requires('users.timfelgentreff.
 	        var cobj = new Constraint(func, this);
 			cobj.allowFailing = true;
 	        cobj.addPrimitiveConstraint(new ReactiveSolver.Constraint(this, cobj, func));
+	        cobj.opts = opts;
```

Comment:
In order to not further complicate the merge with babelsberg/babelsberg.js I would go with _options for now and personally I would rename that opts passed to `always` to options as well. ^^